continuous_integration/recipe/meta.yaml

```diff
@@ -11,8 +11,12 @@ source:
   git_url: ../..
 
 build:
-  noarch: python
   number: {{ GIT_DESCRIBE_NUMBER }}
+  skip: true   # [py2k]
+  entry_points:
+    - dask-sql-server = dask_sql.server.app:main
+    - dask-sql = dask_sql.cmd:main
+  noarch: python
```

Comment:
Should we still build nightlies `noarch` even though the stable releases don't do this?

Reply:
No this should be dropped. It doesn't work with the selector below

Reply:
Will need to use [variants]( https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html ) to construct a build matrix for the Python versions

No easy answers for OS other than including more jobs on other OSes

Am assuming we are the primary users of nightlies. So would use that as a guiding principle until someone else asks for something more

Reply:
Thanks for the link - I'll play around with variants for now, and check in with RAPIDS folk to see if we have any specific needs here.